web/client/utils/__tests__/CoordinatesUtils-test.js

```diff
@@ -847,4 +848,42 @@ describe('CoordinatesUtils', () => {
         expect(res.extent.length).toBe(4);
         expect(res.extent).toEqual([-20026376.39, -20048966.10, 20026376.39, 20048966.10]);
     });
+
+    it('checkIfLayerFitsExtentForProjection out of bounds layer', () => {
+        const geoJson = {
+            bbox: {crs: "EPSG:4326"},
+            "type": "FeatureCollection",
+            "features": [{
+                "type": "Feature",
+                "geometry": {
+                    "type": "Point",
+                    "coordinates": [-150, 94]
+                },
+                "properties": {
+                    "prop0": "value0"
+                }
+            }]
+        };
+        const canFitWithBounds = checkIfLayerFitsExtentForProjection({name: "test", ...geoJson});
+        expect(canFitWithBounds).toBe(false);
```

Comment:
how can this be outside bounds of 4326?

also create some other tests where part of the geometry is outside on each side of the bbox

Reply:
for 4326  this is max extent`[-180, -90, 180, 90]`. so it's outside because the 94 exceeds the maximum of 90